builds/releasenotes.md

```diff
@@ -1,5 +1,18 @@
 udStream Version History
 
+Version 1.2
+  - Added helper when using Dropbox links
+  - Added markdown support
+  - Added scene explorer icons
+  - Added basic fly through recording
+  - Added 31700 and 19920 EPSG codes
+  - (Mac) Fixed an issue with fonts not loading
```

Comment:
This should be `macOS` not `Mac`, we should aim for consistency with the current name that Apple uses.